src/styles.css

```diff
@@ -77,6 +77,13 @@ main{
     border-radius: 4px;
 }
 
+.operations:hover{
```

Comment:
nice

Reply:
It's not a big thing, but probably you want to do a [transition](https://www.w3schools.com/css/css3_transitions.asp). For the animation to be more smooth :)

Reply:
yup